public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -72,17 +76,46 @@ export function createSpanLinkFactory({
       scopedVars = {
         ...scopedVars,
         ...scopedVarsFromSpan(span),
+        ...scopedVarsFromTags(span, traceToProfilesOptions),
       };
       // We should be here only if there are some links in the dataframe
       const fields = dataFrame.fields.filter((f) => Boolean(f.config.links?.length))!;
       try {
+        let profilesDataSourceSettings: DataSourceInstanceSettings<DataSourceJsonData> | undefined;
+        if (traceToProfilesOptions?.datasourceUid) {
+          profilesDataSourceSettings = getDatasourceSrv().getInstanceSettings(traceToProfilesOptions.datasourceUid);
+        }
+        const hasConfiguredPyroscopeDS = profilesDataSourceSettings?.type === 'grafana-pyroscope-datasource';
+        const hasPyroscopeProfile = span.tags.filter((tag) => tag.key === 'pyroscope.profile.id').length > 0;
```

Comment:
What is inside the `pyroscope.profile.id`? Is it the spanID or a full URL or something?

Reply:
It's the spanID.